bruin2015.md

```diff
@@ -0,0 +1,6 @@
+# Add things to LA file
+
+### Things to do in LA
+
+[] take the ferry to Catalina Island
```

Comment:
Cool, I haven't had a chance to do that. Have you gone diving over there? I've seen cool photos of the kelp forests.